fbs/rtpParameters.fbs

```diff
@@ -0,0 +1,91 @@
+
+namespace FBS.RtpParameters;
+
+enum MediaKind : uint8 { ALL = 0, AUDIO, VIDEO }
+enum Type: uint8 { NONE = 0, SIMPLE, SIMULCAST, SVC, PIPE }
+
+// Boolean is a uint8/byte type.
+table Boolean {
+  value:uint8;
+}
+
+table Integer {
+  value:int32;
+}
+
+table IntegerArray {
+  value:[int32];
+}
+
+table Double {
+  value:double;
+}
+
+table String {
+  value:string;
+}
+
+union Value {
+  Boolean,
+  Integer,
+  Double,
+  String,
+  IntegerArray,
+}
+
+table Parameter {
+  name: string;
+  value: Value;
+}
+
+table RtcpFeedback {
+  type:string (required);
+  parameter:string;
+}
+
+table RtpCodecParameters {
+  mime_type:string (required);
+  payload_type:uint8;
+  clock_rate:uint32;
+  channels:uint8;
+  parameters:[Parameter];
+  rtcp_feedback:[RtcpFeedback];
+}
+
+table RtpHeaderExtensionParameters {
+  uri:string (required);
+  id:uint8;
```

Comment:
This is also required

Reply:
Scalars (numbers, booleans) cannot be made required. If not added they will have their default value (0). So we can add default values or add a workaround to make them mandatory.

Reply:
This is on my TODO.

Reply:
Scalar do aways have a value, they don't need (and can't) be marked as required.